cocos/particle/renderer/trail.ts

```diff
@@ -347,7 +347,7 @@ export default class TrailModule {
     public get inited (): boolean {
         return this._inited;
     }
-    private _inited: boolean;
+    private declare _inited: boolean;
```

Comment:
What does declare do? I have forgotten the usage.

Reply:
_inited is assigned in constructor, uses `declare` to avoid assigning twice in the constructor of generated js code.

Reply:
Ah. I remember now.